FiveCalls/FiveCalls/Welcome.swift

```diff
@@ -9,112 +9,127 @@
-            Spacer()
-            Button(action: {
-                onContinue?()
-                dismiss()
-            }) {
-                Text(R.string.localizable.welcomePage2ButtonTitle())
-                    .fontWeight(.semibold)
-                    .foregroundColor(.white)
-                    .font(.system(size: 30))
-            }
-            .padding()
-            .frame(maxWidth: .infinity, minHeight: 73)
-            .background {
-                RoundedRectangle(cornerRadius: 6)
-                    .foregroundColor(Color.fivecallsDarkBlue)
+        ScrollView {
+            VStack {
+                Image(decorative: R.image.fivecallsLogotype)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 292)
+                    .padding(.vertical, 24)
+
+                VStack(alignment: .leading, spacing: 30) {
+                    Label {
+                        VStack(alignment: .leading) {
+                            Text(R.string.localizable.welcomePage1Title())
+                                .fontWeight(.heavy)
+                                .multilineTextAlignment(.leading)
+                            Text(R.string.localizable.welcomePage1Message())
+                                .multilineTextAlignment(.leading)
+                        }
+                        .accessibilityElement(children: .combine)
+                    } icon: {
+                        Image(systemName: "phone.badge.waveform")
+                            .symbolRenderingMode(.palette)
+                            .foregroundStyle(.fivecallsRed, .fivecallsDarkBlue)
+                    }
+                    .labelStyle(CustomLabel(spacing: 20))
+
+                    Label {
+                        VStack(alignment: .leading) {
+                            Text(R.string.localizable.welcomePage2Title())
+                                .fontWeight(.heavy)
+                            Text(R.string.localizable.welcomePage2Message())
+                        }
+                        .accessibilityElement(children: .combine)
+                    } icon: {
+                        Image(systemName: "goforward.5")
+                            .foregroundStyle(.fivecallsDarkBlue)
+                    }
+                    .labelStyle(CustomLabel(spacing: 20))
+
+                    Label {
+                        VStack(alignment: .leading) {
+                            Text(subMessage)
+                                .fontWeight(.heavy)
+                                .opacity(subMessageOpacity)
+                        }
+                        .accessibilityElement(children: .combine)
+                    } icon: {
+                        Image(systemName: "person.2")
+                            .opacity(subMessageOpacity)
+                            .symbolRenderingMode(.palette)
+                            .foregroundStyle(.fivecallsGreen, .fivecallsDarkBlue)
```

Comment:
Can we use `.fivecallsRed` as the secondary color like above? We don't really use green anywhere in the app.